src/libfuncs/felt252_dict_entry.rs

```diff
@@ -207,59 +175,40 @@ pub fn build_get<'ctx, 'this>(
                 &info.branch_signatures()[0].vars[1].ty,
             )?;
 
-        block_is_null.append_operation(cf::br(block_final, &[value_ptr, default_value], location));
+        block_is_null.append_op_result(cf::br(block_final, &[value_ptr, default_value], location))?;
     }
 
     // found block
     {
-        let op = block_is_found.append_operation(llvm::load(
+        let loaded_val_ptr = block_is_found.load(
             context,
+            location,
             result_ptr,
             value_ty,
+            value_layout.align() as i64,
+        )?;
+        block_is_found.append_op_result(cf::br(
+            block_final,
+            &[result_ptr, loaded_value_ptr],
             location,
-            LoadStoreOptions::new().align(Some(IntegerAttribute::new(
-                IntegerType::new(context, 64).into(),
-                value_layout.align() as i64,
-            ))),
-        ));
-        let loaded_value = op.result(0)?.into();
-        block_is_found.append_operation(cf::br(block_final, &[result_ptr, loaded_value], location));
+        ))?;
     }
 
     // construct the struct
 
-    let op = block_final.append_operation(llvm::undef(entry_ty, location));
+    let op = block_final.append_op_result(llvm::undef(entry_ty, location))?;
     let entry_value = op.result(0)?.into();
```

Comment:
here `append_op_result` does internally what the second line does, so op is already `entry_value`